src/components/Beacon/Beacon.js

```diff
@@ -0,0 +1,413 @@
+            >
+              <IconQuestion />
+            </AnimatedDiv>
+          ))
+        }
+      </Transition>
+    </RoundButton>
+  )
+}
+
+ToggleDialogueButton.propTypes = {
+  open: PropTypes.bool.isRequired,
+  onToggle: PropTypes.func.isRequired,
+}
+
+const OptInDialogue = React.memo(({ onClose, onOptIn, ...styles }) => {
+  return (
+    <animated.div {...styles}>
+      <aside
+        css={`
+          background: #fff;
+          border: 1px solid rgba(209, 209, 209, 0.5);
+          box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.15);
+          border-radius: 3px;
+          width: calc(100vw - ${4 * GU}px);
+          position: absolute;
+          bottom: 100%;
+          /* zeroY is relative to beacon container full viewport height
+           * minus 4 GUs (64px) top plus the relative position of zeroY to the
+           * edge of the viewport (60px) plus one GU for margin (16px) */
+          top: calc(-100vh + 140px);
+          /* zeroX is relative to beacon container full viewport width
+           * minus 2 GUs on each side (32px) plus the position of zeroX
+           * relative to the edge of the viewport (60px) */
+          left: calc(-100vw + 92px);
+          z-index: 1;
+          display: flex;
+          flex-direction: column;
+          height: calc(100vh - 60px - ${2 * GU}px - ${4 * GU}px - ${4 * GU}px);
+          overflow: hidden;
+
+          ${breakpoint(
+            'medium',
+            `
+              width: 350px;
+              height: 600px;
+              position: unset;
+            `
+          )}
+        `}
+      >
+        <header
+          css={`
+            border-top-right-radius: 2px;
+            border-top-left-radius: 2px;
+            height: 150px;
+            min-height: 150px;
+            background-color: transparent;
+            background-image: linear-gradient(
+              180deg,
+              ${theme.gradientStart},
+              ${theme.gradientEnd}
+            )};
+            color: ${theme.gradientText}
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex-direction: column;
+          `}
+        >
+          <h2
+            css={`
+              font-size: 18px;
+              line-height: 22px;
+            `}
+          >
+            How can we help?
+          </h2>
+          <img
+            css={`
+              margin-top: ${2.5 * GU}px;
+            `}
+            src={logo}
+            alt="Aragon logo"
+          />
+        </header>
+        <main
+          css={`
+            display: flex;
+            flex: 1;
+            padding: ${2 * GU}px;
+            overflow: auto;
+          `}
+        >
+          <div
+            css={`
+              min-height: 350px;
+              line-height: 22px;
+              font-size: 15px;
+              display: flex;
+              flex-direction: column;
+              flex: 1;
+              justify-content: space-around;
```

Comment:
cc @dizzypaty

Reply:
Ahhhh, now that I re-read this, @sohkai is that pic from an mobile or chrome, because phones actually have the _other-way_ kind of problems (not enough room): https://github.com/aragonone/product/issues/91

Reply:
@AquiGorka It's from Chrome, let's roll this into https://github.com/aragonone/product/issues/91 👍